include/af/graphics.h

```diff
@@ -0,0 +1,88 @@
+/*******************************************************
+ * Copyright (c) 2014, ArrayFire
+ * All rights reserved.
+ *
+ * This file is distributed under 3-clause BSD license.
+ * The complete license agreement can be obtained at:
+ * http://arrayfire.com/licenses/BSD-3-Clause
+ ********************************************************/
+
+#pragma once
+
+#include <af/defines.h>
+#include <af/array.h>
+
+typedef unsigned long long af_window;
+
+typedef struct {
+    int row;
+    int col;
+    const char* title;
+} af_cell;
+
+#ifdef __cplusplus
+#include <utility>
+namespace af
+{
+
+    // FIXME handle copying properly
+class AFAPI Window {
+    private:
+        af_window wnd;
+        /* below attributes are used to track which
+         * cell in the grid is being rendered currently */
+        int _r;
+        int _c;
```

Comment:
They are part of af_window. They are required here as well, how else will a call the following will work.
`Window(1,1).draw(image);`